topics/atom/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: atom-package, atom-packages, atom-theme
+created_by: GitHub
+display_name: Atom
+github_url: https://github.com/atom
+logo: atom.png
+released: February 26, 2014
+short_description: Atom is a modern open source text editor built with web technologies.
```

Comment:
For the short description, I would drop the word 'modern'. I also think [open source](https://en.wikipedia.org/wiki/Open-source_software) is usually written with a hyphen.

> Atom is an open-source text editor built with web technologies.

Reply:
@maxbrunsfeld I personally thought so too, but the [styleguide](https://github.com/github/explore/blob/master/docs/styleguide.md#words-that-can-be-tricky) in this repo says:
> “Open source” is always lowercase, except when at the start of the sentence. You can use it as an adjective without a hyphen, as in “open source project” or “open source software”.

Reply:
I dropped the word 'modern' in efc33c49247a177162f9ff0c00cf9eb75baf43e5, thanks.

Reply:
> I also think open source is usually written with a hyphen.

@maxbrunsfeld v. @MikeMcQuaid, go! 💪 💥 😀

Reply:
@cheshire137 @maxbrunsfeld I have little preference but may as well follow the style guide 😆